specification/ai/Azure.AI.Projects/evaluations/models.tsp

```diff
@@ -12,6 +12,64 @@ using TypeSpec.Versioning;
 
 namespace Azure.AI.Projects;
 
+
+@doc("Labeled results per row.")
+@added(Versions.v2025_05_15_preview)
+@removed(Versions.v1)
+model LabeledRows {
+  @doc("The label associated with the evaluation.")
+  label: string;
+
+  @doc("Detailed reasoning or explanation for the evaluation.")
+  reasoning: string;
+
+  @doc("The version identifier for the evaluation model.")
+  version: string;
+}
+
+@doc("Summary across rows for evaluation.")
+@added(Versions.v2025_05_15_preview)
+@removed(Versions.v1)
+model MetricSummary extends Results {
```

Comment:
Where do we use `MetricSummary`?

Reply:
with this design I'm saying it will be one of the result types returned in the array, but actually thinking about it again I think we agreed if we always commit to always returning it, then we should have it be a dedicated parameter

Reply:
Sounds good! I asked because `outputSummary?: MetricSummary;` was not in the TypeSpec. I found you added it now, it's good enough